.github/workflows/scripts/code_coverage_disable_check.py

```
@@ -0,0 +1,161 @@
+                and not item.endswith(".spec.ts")
+            ):
+                if has_code_coverage_disable(item):
+                    print(
+                        f"""File {item} contains code coverage disable statement. Please remove it and add the appropriate tests."""
+                    )
+                    code_coverage_found = True
+
+    return code_coverage_found
+
+
+def arg_parser_resolver():
```

Ruff (0.8.2)

Missing return type annotation for public function `arg_parser_resolver`

(ANN201)